modules/cmpapi/src/CmpApi.ts

```diff
@@ -13,16 +13,20 @@ export class CmpApi {
   /**
    * @param {number} cmpId - IAB assigned CMP ID
    * @param {number} cmpVersion - integer version of the CMP
+   * @param {number} tcfPolicyVersion - default policy version to use if not provided by a consent string
    * @param {boolean} isServiceSpecific - whether or not this cmp is configured to be service specific
    * @param {CustomCommands} [customCommands] - custom commands from the cmp
    */
-  public constructor(cmpId: number, cmpVersion: number, isServiceSpecific = false, customCommands?: CustomCommands) {
+  public constructor(cmpId: number, cmpVersion: number, tcfPolicyVersion: number, isServiceSpecific = false, customCommands?: CustomCommands) {
```

Comment:
This seems like too much of a breaking change (to me), forcing all the CMP's to update their constructor. Should we just default it to the current value and provide a settor?

Reply:
Also, this should not be settable because it describes the protocol that this CMP API conforms to and not the TCF Policy version of the consent string or the TCF Policy version of the GVL.

Reply:
This should be removed in the latest push. Added initially for completeness/future proofing, but I agree with above.